programs/bench.h

```diff
@@ -19,46 +19,150 @@ extern "C" {
 #define ZSTD_STATIC_LINKING_ONLY   /* ZSTD_compressionParameters */
 #include "zstd.h"     /* ZSTD_compressionParameters */
 
+typedef enum {
```

Comment:
It's probably better to introduce these types closer to where they are actually needed.